### PR TITLE
test: improve unit test coverage of seata-tm moudle

### LINKED ISSUE #6837: Improve test coverage
Improve test coverage

### PR DESCRIPTION
**Pull request by YvCeung**
- [x] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
Add unit test cases to complete the unit testing objectives for the `seata-tm` module.
增加单元测试用例，完成 seata-tm模块的单元测试目标

### Ⅱ. Does this pull request fix one issue?

fix #6837 

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
Not applicable.
不适用

### Ⅳ. Describe how to verify it
Run `mvn clean test`

### CODE REVIEW

File: tm/src/test/java/org/apache/seata/tm/DefaultTransactionManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seata.tm;
+
+import org.apache.seata.core.exception.TransactionException;
+import org.apache.seata.core.model.GlobalStatus;
+import org.apache.seata.core.protocol.ResultCode;
+import org.apache.seata.core.protocol.transaction.*;
```
Comment (xjlgod) [inline tm/src/test/java/org/apache/seata/tm/DefaultTransactionManagerTest.java]: do not use .*

Comment (YvCeung) [reply tm/src/test/java/org/apache/seata/tm/DefaultTransactionManagerTest.java]: yeah，this is not a good habit,i have fix it ,thanks.

**Review by funky-eyes — APPROVED** [review]
LGTM

**Comment by funky-eyes** [comment]
Please send your DingTalk ID to `jianbin@apache.org`, and I will invite you to join the Seata developers group.

**Pull request merged**